types/block.go

```diff
@@ -1039,124 +1027,17 @@ type Data struct {
 
 // Hash returns the hash of the data
 func (data *Data) Hash() tmbytes.HexBytes {
```

Comment:
this implementation was updated to conform to https://github.com/tendermint/tendermint/blob/main/types/block.go#L1004-L1013